src/DataFixtures/VehicleFixtures.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\DataFixtures;
+
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+use App\Entity\Vehicle;
+
+class VehicleFixtures extends Fixture
+{
+    public function load(ObjectManager $manager)
+    {
+        //Add 5 vehicles random with fixtures
+        for ($i = 0; $i < 5; $i++) {
+            $vehicle = new Vehicle();
+            $vehicle->setName('Peugeot');
+            $vehicle->setDescription('jdsbcsdhjbfdsfdhb');
```

Comment:
Nice description =)

Reply:
Nice description c'est à dire ?

Reply:
As it will be the description of the first and only starting vehicule in the dev database you could have been more inventive =)